docs/docs/usage/scaladoc/scaladocDocstrings.md

```diff
@@ -0,0 +1,188 @@
+
+```scala
+scalacOptions in (Compile, doc) ++= Seq(
+  "-groups"
+)
+```
+Each section should have a single-word identifier that is used in all of these tags, shown as <group> below. By default, that identifier is shown as the title of that documentation section, but you can use @groupname to provide a longer title.
+
+Typically, you should put @groupprio (and optionally @groupname and @groupdesc) in the Scaladoc for the package/trait/class/object itself, describing what all the groups are, and their order. Then put @group in the Scaladoc for each member, saying which group it is in.
+
+Members that do not have a `@group` tag will be listed as “Ungrouped” in the resulting documentation.
+
+- `@group <group>` - mark the entity as a member of the `<group>` group.
+- `@groupname <group> <name>` - provide an optional name for the group. `<name>` is displayed as the group header before the group description.
+- `@groupdesc <group> <description>` - add optional descriptive text to display under the group name. Supports multiline formatted text.
+- `@groupprio <group> <priority>` - control the order of the group on the page. Defaults to 0. Ungrouped elements have an implicit priority of 1000. Use a value between 0 and 999 to set a relative position to other groups. Low values will appear before high values.
+
+### Other tags
+
+- `@author` provide author information for the following entity
+- `@version` the version of the system or API that this entity is a part of.
+- `@since` like `@version` but defines the system or API that this entity was first defined in.
+- `@deprecated` marks the entity as deprecated, providing both the replacement implementation that should be used and the version/date at which this entity was deprecated.
+- `@syntax <name>` let you change the parser for docstring. The default syntax is markdown, however you can change it using this directive. Currently available syntaxes are `markdown` or `wiki`, e. g. `@syntax wiki`
+
+### Macros
+
+- `@define <name> <definition>` allows use of $name in other Scaladoc comments within the same source file which will be expanded to the contents of `<definition>`.
+
+If a comment is not provided for an entity at the current inheritance level, but is supplied for the overridden entity at a higher level in the inheritance hierarchy, the comment from the super-class will be used.
+
+Likewise if `@param`, `@tparam`, `@return` and other entity tags are omitted but available from a superclass, those comments will be used.
+Explicit
+
+For explicit comment inheritance, use the @inheritdoc tag.
+Markup
+
+It is still possible to embed HTML tags in Scaladoc (like with Javadoc), but not necessary most of the time as markup may be used instead.
+
+### Markup
+
+Scaladoc provides two syntax parsers: `markdown` (default) or `wikidoc`.
+
+#### Markdown
+
+Markdown uses commonmark flavour to parse docstring, however it can understand `wikidoc` links.
```

Comment:
1. i wouldn't mention wikidoc here at all, line 120 above already did.
2. could you include a link to a recent commonmark spec?

Reply:
1. The clue here is that you can write markdown code with wikidoc links, e. g. `This is my *markdown* with wikidoc link to [[ my.Class ]]`. If this sentence is confusing, I can elaborate.
2. Will fix

Reply:
this comment seems to suggest that one must choose either markdown or the old way, not both, and the only exception is those links.

if that's not true, why do you need to call out the links specifically, if one can mix and match anything with anything?

Reply:
I'm not sure whether you talk about outdated or new version of this sentence. 

As it comes to language for docstring styling, you have to choose either `Wikidoc` or `Markdown` (at least single docstring must be fully markdown or fully wikidoc), however we added extension to markdown syntax so you can use `Wikidoc` links while using `Markdown` regular elements like bold with asterisks. It is quite useful, because these links are more intelligent than regular markdown links and let you link classes etc. by their fully qualified name, so you don't have to know the absolute path of these elements.

I thought one would not be aware of this extension so this is what I meant saying `Markdown uses commonmark flavour with extension of wikidoc links for referencing convenience.`

Reply:
Wikidoc syntax is legacy syntax used by old scaladoc. I think it's spec can be found [here](https://www.wikidoc.org/index.php/Cheat_Sheet). The 3 curly braces was a way to declare code blocks in `Scala2` (I'm not sure is it official spec or just addition by Scala2 scaladoc creators). In new Scaladoc we assume 3 curly braces syntax as part of `Wikidoc` and 3 backticks as `Markdown`.

But...

I must have missed Alekasnder's [PR](https://github.com/lampepfl/dotty/pull/11833) that implements curly braces as code blocks valid syntax even if Markdown is used. I guess I need to update the docs then.